code/modules/surgery/bodyparts/bodyparts.dm

```diff
@@ -182,14 +184,15 @@
 	//We've dealt the physical damages, if there's room lets apply the stamina damage.
 	var/current_damage = get_damage(TRUE)		//This time around, count stamina loss too.
 	var/available_damage = max_damage - current_damage
-	stamina_dam += round(CLAMP(stamina, 0, min(max_stamina_damage - stamina_dam, available_damage)), DAMAGE_PRECISION)
+	var/applied_damage = min(max_stamina_damage - stamina_dam, available_damage)
+	stamina_dam += round(CLAMP(stamina, 0, applied_damage), DAMAGE_PRECISION)
```

Comment:
Variable names are very confusing. `available_damage` is the "room" in terms of `current_damage`, and `applied_damage` is the "room" considering both `current_damage` and `stamina_dam` and their respective limits(`max_damage`, `max_stamina_damage`).

Make a var like `var/available_stamina_damage = min(max_stamina_damage - stamina_dam, max_damage - current_damage)` or inline the entire thing.